static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,32 @@ define([
         }
     }
 
+    function initMostPopularCard(el) {
+        var height;
+        var $adSlot = $(el);
+        var $mostPopTabs;
+
+        if ($adSlot.hasClass('ad-slot--mostpop')) {
+            fastdom.read(function () {
+                height = $adSlot.dim().height;
+                $mostPopTabs = $('.js-most-popular-footer .tabs__pane');
+            });
+
+            fastdom.write(function () {
+                $mostPopTabs.css('height', height);
+            });
+        }
+
+        createToggle(el);
+    }
+
     CommercialComponent.prototype.postLoadEvents = {
         bestbuy: function (el) {
             new Tabs().init(el);
         },
         capi: createToggle,
         capiSingle: createToggle,
-        paidforCard: createToggle
+        paidforCard: initMostPopularCard
```

Comment:
Even better! This way the logic inside each callback is completely sealed from the rest. We should definitely build a library of higher-order functions like this one, it would help write much more readable code.